scripts/transcript_summarizer_aws_lambda.py

```diff
@@ -1,26 +1,32 @@
 import json
-
 import boto3
-
-destination_bucket = "transcriptslangchain"
+import os
+destination_bucket = os.environ['destination_bucket']
```

Comment:
Define an environment variable for the bucket name in /Users/oshoma/dev/sherpa/src/sherpa_ai/config/__init__.py  and use it via sherpa_ai.config.